src/GitVersion.Core/BuildAgents/BitBucketPipelines.cs

```diff
@@ -0,0 +1,65 @@
+using GitVersion.Logging;
+using GitVersion.OutputVariables;
+
+namespace GitVersion.BuildAgents;
+
+public class BitBucketPipelines : BuildAgentBase
+{
+    public const string EnvironmentVariableName = "BITBUCKET_WORKSPACE";
+    public const string BranchEnvironmentVariableName = "BITBUCKET_BRANCH";
+    public const string TagEnvironmentVariableName = "BITBUCKET_TAG";
+    public const string PullRequestEnvironmentVariableName = "BITBUCKET_PR_ID";
+    private string? file;
+
+    public BitBucketPipelines(IEnvironment environment, ILog log) : base(environment, log) => WithPropertyFile("gitversion.env");
```

Comment:
Other CI Systems use the filename `gitversion.properties`. Is there a good reason to use `gitversion.env` rather than remaining consistent?

Reply:
Only because we're not writing out the properties _per se_, but exporting each individual property as an environment variable. I'm guessing that the name change won't make any material difference, it should still be consumed the same:

```yml
- step:
  name: Version
  script:
    - source gitversion.properties
    - echo Building version $GITVERSION_FULLSEMVER
```

I am happy to make the change if that's what people would like.